src/otrs-popup/otrs-popup.controller.js

```diff
@@ -19,11 +20,14 @@ export default /* @ngInject */ function (
   $translate,
   OvhApiMe,
   OvhApiMeVipStatus,
-  OvhApiProductsAapi,
+  OvhApiService,
   OvhApiSupport,
   OtrsPopup,
   OtrsPopupInterventionService,
   OtrsPopupService,
+  OTRS_POPUP_API_EXCLUDED,
+  OTRS_POPUP_API_EXTRAS_ENDPOINTS,
+  OTRS_POPUP_API_ALIASES,
```

Comment:
You could import your constants directly, no need to inject them as angular dependency. What do you think ?

Reply:
This component is rewritten (thanks @FredericEspiau) so I think we can stick with it.

ref: https://github.com/ovh-ux/manager/tree/feat/support

Reply:
If by rewriting you mean deleted then yes 🎉